extensions/eo/README.md

```diff
@@ -52,24 +60,24 @@ The band's common_name is the name that is commonly used to refer to that band's
 properties. The table below shows the common name based on the average band range for the band
 numbers of several popular instruments.
 
-| Common Name | Band Range (μm) | Landsat 5/7 | Landsat 8 | Sentinel 2 | MODIS |
-| ----------- | --------------- | ----------- | --------- | ---------- | ----- |
-| coastal     | 0.40 - 0.45     |             | 1         | 1          |       |
-| blue        | 0.45 - 0.50     | 1           | 2         | 2          | 3     |
-| green       | 0.50 - 0.60     | 2           | 3         | 3          | 4     |
-| red         | 0.60 - 0.70     | 3           | 4         | 4          | 1     |
-| yellow      | 0.58 - 0.62     |             |           |            |       |
-| pan         | 0.50 - 0.70     | 8 (*L7 only*) | 8       |            |       |
-| rededge     | 0.70 - 0.75     |             |           |            |       |
```

Comment:
Sentinel-2 calls band 7 (which goes up to 0.782μm) a red edge band, so let's expand the range slightly.

Reply:
Are there any "official"/scientific values we could relate to here?

Reply:
There may be, but I the approach we've take is the right one -- look at what range the values that each sensor calls "red", etc and make that the range for common_name red.